ts/Core/Series/DataLabel.ts

```diff
@@ -564,6 +564,11 @@ namespace DataLabel {
         fireEvent(this, 'drawDataLabels');
 
         if (series.hasDataLabels?.()) {
+            const nullLabel = (
+                seriesOptions?.nullInteraction &&
+                'Null'
+            );
```

Comment:
I see this part was removed, then 8 diffs appeared, and a new commit named "Accommodating map series" re-introduced it. What happened? When I remove the `nullLabel` logic, I have no visual diffs in the maps section locally. Also, if the problem applies only to the map series, we should address it in the map code, not the core, for example by setting a default `dataLabels.nullFormatter`.

Reply:
Oh sorry - this was supposed to be gone from the PR as was the logic using it.